### ISSUE TITLE
Update Argo Blockchain Information (Ticker & BTC HODL)

### ISSUE THREAD

**Issue by tddivine** [nvk/bitcointreasuries.NET#316]
Hello,

I am requesting an update to Argo Blockchain's information, two items in particular:

Ticker: In Sept 2021, Argo listed its ADRs on the Nasdaq under the ticker "ARBK" (source: https://polaris.brighterir.com/public/argo_blockchain/news/rns/story/rdg8y8r) 
BTC HODL: As of 11/30/21, Argo's BTC HODL was 2,317 (source: https://polaris.brighterir.com/public/argo_blockchain/news/rns/story/rg8kn3w)

Thank you.

**Comment by tddivine** [comment]
Hi @nvk, I am checking in to see if this can please be updated?

**Comment by tddivine** [comment]
Can you please update the BTC holdings as well to 2,317?

**Comment by nvk** [comment]
What's the basis cost?

**Comment by tddivine** [comment]
I don't believe we have disclosed that publicly. Is that required? Can you update the BTC holdings and current market value and keep the cost basis as a question mark?

**Comment by tddivine** [comment]
@nvk, I confirmed that we do not disclose the cost basis. I see that you've updated the BTC holding, thank you. One more item that I noticed. The market cap ($384 million as of 1/7/22 at 12:22pm CT) appears to be our market cap in GBP, not USD. Our market cap in USD is approximately $520 million. Can this be corrected as well?

Thanks.
